test_scripts/Polices/build_options/031_ATF_P_TC_PTU_Retry_Sequence_Retry_Timeout_Computation_HTTP.lua

```diff
@@ -48,10 +49,10 @@ local f_name = os.tmpname()
 local ptu
 local sequence = { }
 local accuracy = 5
-local r_expected = { 11, 22, 33 }
+local r_expected = { 30, 31, 62, 93, 124} -- 155 is for finish retry sequence
 local r_actual = { }
-local attempts = 15
-local timestamps = { }
+local attempts = ( 30 + 31 + 62 + 93 + 124 )
+local timestamps = { 0, 0, 0, 0, 0, 0 }
```

Comment:
@istoimenova, By using predefined values in _timestamps_ lines 210 and 219 have to be updated since insert function is used there.
Or other option is not to use predefined values here.